Source/ASCollectionNode.h

```diff
@@ -291,6 +315,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)waitUntilAllUpdatesAreProcessed;
 
+/**
+ *  Returns YES if the ASCollectionNode contents are completely synchronized with the underlying collection-view layout.
+ */
+@property (nonatomic, readonly) BOOL isSynchronized;
```

Comment:
`synchronized` with an `isSynchronized` getter?

Reply:
@Adlai-Holler This is an interesting case because it's a read-only property, so only the getter applies. I'm happy to change it in the name of convention, though it felt a bit unusual to write!